test/browser/map.test.jsx

```diff
@@ -129,6 +131,22 @@ describe("Map.MarkerPopup", function() {
     findButtons(popup).length.should.equal(2);
```

Comment:
Same as https://github.com/mozilla/teach.webmaker.org/pull/965/files#r31479842 - could we also check to see if a "pending" or "denied" label is presented?